Gems/ROS2/Code/Source/ROS2GemUtilities.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2/ROS2GemUtilities.h"
```

Comment:
This include for the ROS2GemUtilities.h header in the Include directory should use angle brackets.
```suggestion
#include <ROS2/ROS2GemUtilities.h>
```